hoplite-core/src/main/kotlin/com/sksamuel/hoplite/decoder/dates.kt

```diff
@@ -65,3 +62,27 @@ class InstantDecoder : NonNullableDecoder<Instant> {
     else -> ConfigFailure.DecodeError(node, type).invalid()
   }
 }
+
+class YearDecoder: NonNullableDecoder<Year> {
```

Comment:
The only other thing needed is to register them in META-INF/services

Reply:
Already done: https://github.com/sksamuel/hoplite/blob/5927b10990b6cffd65c8fa609da54bfb253a5942/hoplite-core/src/main/resources/META-INF/services/com.sksamuel.hoplite.decoder.Decoder

Reply:
Must have missed it !